content/docs/rfcs/17/README.md

```diff
@@ -203,6 +210,13 @@ With a reasonable approximation of the preceding values, one can set  `max_epoch
 `Network_Delay` and `Clock_Asynchrony` MUST have the same resolution as  `period` .
 By this formulation,  `max_epoch_gap`  indeed measures the maximum number of `epoch`s that can elapse since a message gets routed from its origin to all the other peers in the network.
 
+`acceptable_root_window_size` depends upon the underlying chain's average blocktime, and `Network_Delay`
+
+`acceptable_root_window_size` can be set as $acceptable_root_window_size=block_time/Network_Delay$
+
+By this formulation, `acceptable_root_window_size` will provide an approximation of how many roots can be acceptable by a routing peer.
+However, the peer can be generous with this window size if they desire enhanced slashing opportunities.
```

Comment:
If different peers within the same spam-protected pubsub/content topic choose different window sizes, wouldn't this affect message routing quite significantly? Should this be a property agreed upon for the entire spam-protected domain?

Reply:
Indeed, this would affect message routing. However, due to the slashing incentive, peers would be attempting to maximize the number of roots they store locally (as long as the roots are within the maximum epoch time, otherwise the message is invalid nonetheless). Eventually, we will see all nodes converge upon a single value to maximize opportunities to slash.

Additionally, There could be a scenario in which a malicious actor would set the `acceptable_root_window_size` to 0, but that would not have an impact on a well connected network, unless a majority attack is carried out, which is out of scope for this RFC.